mlibrary/lib.umich.edu
# Remove unnecessary aria-labels

# Overview
As a continuation of the baseline accessibility review, this resolves the issue with unnecessary aria-labels. Three locations were identified where aria-labels were being improperly used.

1. Homepage hours summary panel: `<span aria-label="TODAY: 8am to 12am" class="css-tqvqi2">TODAY: 8am - 12am</span>`
2. Location details page header section: `<header aria-label="Location description">`
3. Locations and hours page

These aria-labels were removed.

> This pull request fixes [WEBSITE-133](https://mlit.atlassian.net/jira/software/projects/WEBSITE/boards/27?selectedIssue=WEBSITE-133).

##NOTE
On VoiceOver the headings on the tables for the Locations and Hours view are read twice despite only having one label. I am not 100% sure why it does this but the current solution works the best (and better than it did before) between NVDA and VO.

## Testing
Run the app and check the Plan your visit section on the homepage. Check that the `aria-label` attributes are no longer on the hours summary.

Check a location details page (/locations-and-hours/hatcher-library) and make sure the `<header>` no longer has an aria-label.

Check the Locations and Hours page and make sure the aria-labels are removed: /locations-and-hours/hours-view

- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
